components/front-page/LearnSection.tsx

```diff
@@ -1,32 +1,49 @@
 import { motion } from "framer-motion";
+import Image from "next/image";
 
 interface LearnCardProps {
   tag: string;
   title: string;
   description: string;
   link: string;
-  className?: string;
+  img: string;
+  classes?: string;
 }
 
 const LearnCard = ({
   tag,
   title,
   description,
-  className,
+  classes,
   link,
+  img,
 }: LearnCardProps) => {
   return (
     <motion.div
-      whileHover={{ scale: 1.03 }}
+      whileHover={{
+        scale: 1.03,
+        boxShadow: "0px 10px 20px 5px rgba(0,0,0,0.25)",
+      }}
       whileTap={{ scale: 1 }}
-      className={`${className} min-w-[220px] w-full h-[126px] rounded-[10px] p-[15px]`}
+      className={`${classes} w-full rounded-[10px] p-[15px] pr-[20px]`}
```

Comment:
shouldn't the `${classes}` go at the end of the string so it's possible to override default styles e.g. setting rounded corners to 0, or changing padding. Otherwise we'd need to use `important`